### PR TITLE
[PDE-198] job api geographic information

### PR DESCRIPTION
**Pull request by leo-oliveira**
- Created new property `Geolocation` of type `JobGeoLocationResponse` in JobResponse  which contains information about the job geolocation.

### CODE REVIEW

File: eRecruiter.Api/Responses/Job/JobGeoLocationResponse.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace eRecruiter.Api.Responses
+{
+    public class JobGeoLocationResponse
```
Comment (bwolkerstorfer) [inline eRecruiter.Api/Responses/Job/JobGeoLocationResponse.cs]: As we are going to use in other responses `GeoLocationResponse` would be more suitable.

File: ChangeLog.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # eRecruiter API ChangeLog
 
+## 1.45.0
+- New property `Geolocation` of type `JobGeoLocationResponse` in JobResponse  which contains information about the job geolocation.
```
Comment (bwolkerstorfer) [inline ChangeLog.md]: Property is named `GeoLocation` in the class please correct to match the name in the class.

File: eRecruiter.Api/Responses/Job/JobGeoLocationResponse.cs (hunk 1 above)
Comment (leo-oliveira) [reply eRecruiter.Api/Responses/Job/JobGeoLocationResponse.cs]: done.

File: ChangeLog.md (hunk 2 above)
Comment (leo-oliveira) [reply ChangeLog.md]: done.

**Pull request merged**